aleph-client/src/pallets/multisig.rs

```diff
@@ -1,20 +1,45 @@
+use std::collections::HashSet;
+
+use anyhow::{anyhow, ensure, Result as AnyResult};
+use codec::{Decode, Encode};
 use primitives::{Balance, BlockNumber};
+use sp_core::blake2_256;
+use sp_runtime::traits::TrailingZeroInput;
 
 use crate::{
-    api, api::runtime_types, sp_weights::weight_v2::Weight, AccountId, BlockHash,
-    SignedConnectionApi, TxStatus,
+    account_from_keypair, aleph_runtime::RuntimeCall, api, api::runtime_types,
+    sp_weights::weight_v2::Weight, AccountId, BlockHash, ConnectionApi, SignedConnectionApi,
+    TxStatus,
 };
 
 pub type CallHash = [u8; 32];
-pub type Call = Vec<u8>;
-pub type Timepoint = api::runtime_types::pallet_multisig::Timepoint<BlockNumber>;
+pub type Call = RuntimeCall;
```

Comment:
those type aliases should be as well commented I think, otherwise there will be warns